### ISSUE TITLE
Online Co-op error: Creature.instantiatedTemporaryItem needs to be null first before calling EquipTemporaryItem

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#668]
Multiple auto bug reports, v0.1.8a

Feedback IDs:
- 20201120-VXBCR
- 20201119-8YT28
- 20201119-56LNY
- 20201119-D3V26

>I'm in a online game, in the residenz and holding already the golden key and trying to chance the avatar at the mirror.
It's giving me the F2 notice, but I can still play on. No crash...

```
9:42:01, Frame 6943, ERROR »  ERROR: Creature.instantiatedTemporaryItem needs to be null first before calling EquipTemporaryItem
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>


GameStates: InitialisingAsNetworkHost, RompingThroughDungeon, UsingQuasiModalCanvas
GameSetupMode: NetworkCoop
Players: 1
DungeonRoomTransitionerState: Null
Room: 0 / 0
RoomModifiers: None
YouHaveDiedState: Null
SystemPlayer InputDevice: XInput Controller
HumanPlayer 1 InputDevice: XInput Controller
```

To try and reproduce: Online Co-op, Home room, hold key, try to use Magic Mirror

---

> Changing character when holding key does not transfer the key to the new character and then the key no longer works and effectively breaks the level.

**Comment by ChrisNZL** [comment]
Further auto bug report feedback IDs – v0.2.0:
- 20201205-MWYZL

**Comment by ChrisNZL** [comment]
Fixed in 0.2.1.

Fix is client-side only; did not require server modification.
